src/index.js

```diff
@@ -51,6 +52,7 @@ module.exports = {
       return [...fullCorvidTypes.BASE, ...fullCorvidTypes.PUBLIC];
     }
   },
+  wixModulesNames,
```

Comment:
consider changing to `getWixModulesList` more general API

Reply:
Done